packages/web-platform/web-tests/rspack.config.js

```diff
@@ -203,7 +203,7 @@ const config = {
       });
       return middlewares;
     },
-    watchFiles: ['./dist/**/*', './node_modules/@lynx-js/**/*'],
+    watchFiles: ['./node_modules/@lynx-js/**/*'],
```

Comment:
Can we disable `watchFiles` on CI?

And it seems good to have `dist/**` in `watchFiles` for local debugging.